app/services/authentication/omniauth_service.rb

```diff
@@ -217,21 +248,24 @@ def build_omniauth_hash_to_user_attributes
       info = auth_hash[:info]
 
       attribute_map = {
-        login: info[:email],
+        login: info[:login] || info[:email],
```

Comment:
Just wondering if this might lead to unexpected consequences. This means that for existing users their login might be changed from their email address to what ever login the omniauth provider returns.

It probably doesn't matter since the login is not used for logging in (since it's omniauth) and people are mentioned using their full names too. So I suppose this is ok? Just double checking.

I wonder if we should make this dependent on `Setting.email_login` and still use email here if that is true.